src/api/compendia.js

```diff
@@ -0,0 +1,33 @@
+import { Ajax } from '../common/helpers';
+
+export async function fetchCompendiaData(additionalFilters = {}, token) {
+  try {
+    const filters = {
+      ...{
+        latest_version: true,
+        ordering: 'primary_organism__name',
+        limit: 1000,
+      },
+      ...additionalFilters,
+    };
+
+    const values = Object.entries(filters).map(([key, value]) => {
+      return `${key}=${value}`;
+    });
+    const endpoint = `/v1/compendia/?${values.join('&')}`;
```

Comment:
You can also pass `filters` as the second parameter of `Ajax.get`, and it'll do something like this.